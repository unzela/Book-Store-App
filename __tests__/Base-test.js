import React from 'react';
import renderer from 'react-test-renderer';
import BaseHome from './../src/base';

test('BaseHome renders', () => {
  const tree = renderer.create(<BaseHome />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('BaseHome Snapshot',()=>{
  renderer.create(<BaseHome/>);

});