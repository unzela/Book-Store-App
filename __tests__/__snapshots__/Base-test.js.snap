// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseHome renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "lightgrey",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fb5b5a",
        "fontSize": 50,
        "fontWeight": "bold",
        "marginBottom": 40,
      }
    }
  >
    Welcome! 
  </Text>
  <Text
    style={
      Object {
        "color": "#fb5b5a",
        "fontSize": 50,
        "fontWeight": "bold",
        "marginBottom": 40,
      }
    }
  >
    Books App
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 10,
        "marginTop": 40,
        "opacity": 1,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
        }
      }
    >
      LOGIN
    </Text>
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 10,
        "marginTop": 40,
        "opacity": 1,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
        }
      }
    >
      Register
    </Text>
  </View>
</View>
`;
